library/hls/src/main/java/com/google/android/exoplayer2/source/hls/playlist/HlsMasterPlaylist.java

```diff
@@ -201,23 +227,25 @@ public Rendition(@Nullable Uri url, Format format, String groupId, String name)
    * @param sessionKeyDrmInitData See {@link #sessionKeyDrmInitData}.
    */
   public HlsMasterPlaylist(
-      String baseUri,
-      List<String> tags,
-      List<Variant> variants,
-      List<Rendition> videos,
-      List<Rendition> audios,
-      List<Rendition> subtitles,
-      List<Rendition> closedCaptions,
-      Format muxedAudioFormat,
-      List<Format> muxedCaptionFormats,
-      boolean hasIndependentSegments,
-      Map<String, String> variableDefinitions,
-      List<DrmInitData> sessionKeyDrmInitData) {
+          String baseUri,
+          List<String> tags,
+          List<Variant> variants,
+          List<Variant> iFrameVariants,
+          List<Rendition> videos,
+          List<Rendition> audios,
+          List<Rendition> subtitles,
+          List<Rendition> closedCaptions,
+          Format muxedAudioFormat,
+          List<Format> muxedCaptionFormats,
+          boolean hasIndependentSegments,
+          Map<String, String> variableDefinitions,
+          List<DrmInitData> sessionKeyDrmInitData) {
     super(baseUri, tags, hasIndependentSegments);
```

Comment:
Sorry, looks like I've missed the spacing default.  It should be 2 spaces for indent I assume?